.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: []
+    branches: [ master ]
```

Comment:
Hm.. Currently the CI runs are not enabled for pull requests. Was this intentional, or?